From f10e3e279ee28cd477d3bcb0910353afcc58699c Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Thu, 4 Jan 2024 17:31:18 +0100
Subject: [PATCH 0928/1141] arm64: dts: st: add wake-gpios for pcie in
 stm32mp257f-ev1

Add wake-gpios to manage wake# signal for RC mode

Deasserting the input wake# triggers an irq used to resume
the system when in low power mode.

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: I246e7537659d433d2e126245dd160b7299bfb7be
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/350790
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index d03a69297b73..11ecc9f69347 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -559,6 +559,8 @@ &pcie_rc {
 	pinctrl-1 = <&pcie_init_pins_a>;
 	pinctrl-2 = <&pcie_sleep_pins_a>;
 	reset-gpios = <&gpioj 8 GPIO_ACTIVE_LOW>;
+	wakeup-source;
+	wake-gpios = <&gpioh 5 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
 	status = "okay";
 };
 
-- 
2.39.2

